wiring/src/spark_wiring_rgb.cpp

```diff
@@ -61,4 +60,10 @@ void RGBClass::brightness(uint8_t brightness, bool update)
         LED_On(LED_RGB);
 }
 
+void RGBClass::attachHandler(led_update_handler_fn fn)
```

Comment:
I suggest adding an optional void* data parameter so that users can have an optional data item passed back to their handler.